Overview.bs

```diff
@@ -1849,10 +1841,18 @@ to give the smallest encodings for most unicode code point sets typically encoun
   ```
 </div>
 
-### URI Templates ### {#uri-templates}
+### URL Templates ### {#url-templates}
+
+URL templates [[!rfc6570]] are used to convert numeric or string IDs into URL strings where patch files are located.
```

Comment:
I'd prefer leaving "URI" here and then clarify that the template must expand into a URL

Reply:
Sounds good, fixed.